openapiart/bundler.py

```diff
@@ -1060,6 +1060,68 @@ def _generate_value_schema(
                 metric_tags_schema_name
             ] = metric_tags_schema
 
+        if "features" in xpattern and "random" in xpattern["features"]:
+            if xpattern["format"] in ["integer", "ipv4", "ipv6", "mac"]:
+                random_pattern_name = "{}.Random".format(schema_name)
```

Comment:
Should we specify the behavior of the implementation for cases where the range of random values is less than the range of the requested format? In particular some implementations may not be able to randomize the whole ipv6 field.

Reply:
So should be specify that in the description of ipv6 fields in random ?
what should we specify as max for ipv6 ?

Reply:
If range of random values is less than the requested format, then implementation will raise error to reduce requested range. We can add this support once we need to support IPv6 / other format, where such situation can arise.

Reply:
A range of less than the requested format is a valid and common use case, so we must allow it. But we should change min and max to accommodate 128-bit min and max values. For UHD, the range of random numbers (max-min+1) must fit within a uint32, but we could conceivably allow a 128-bit min value, though we are currently implemented as a 32-bit min value.

Reply:
But since this review only affects openapiart it's fine for now.